src/index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/8f76becb1e.js" crossorigin="anonymous"></script>
+    <title>JavaScript Capstone Project</title>
+</head>
+<body>
+    <header>
+        <div class="logo">
+            <a href="index.html">
+                Logo
+            </a> 
+        </div>
+        <nav>
+            <a class="my-movies shows" href="index.html"> Shows</a>
+            <a href=""> Categories </a>
+            <a href=""> Trending </a>
+        </nav>
+    </header>
```

Comment:
[Optional] Don't you think the navbar will look better if you add a background color to it? Currently, it does not look well when the background content has some color.

![image](https://user-images.githubusercontent.com/34779928/197282711-3159d515-973d-4d2c-bda6-fe6cb69054ed.png)